src/strings/PowerFxResources.en-US.resx

```diff
@@ -6538,6 +6538,10 @@
     <value>As is not permitted in this context</value>
     <comment>{Locked=As} This is an error message that shows up when the As keyword is used but is not valid</comment>
   </data>
+  <data name="ErrorResource_ErrUntypedObjectScope_ShortMessage" xml:space="preserve">
+    <value>Untyped objects cannot be used as the first argument to functions which support record scopes.</value>
+    <comment>This error message shows up when using untyped objects as the first arugment to functions with lambda overloads</comment>
```

Comment:
Do we need to lock Untyped Object? Do we translate it other places?

Reply:
It is rare, untyped object, UntypedObject, and ParseJSON do not appear in the canvas C# resource strings. They do appear in the TypeScript resource strings. ParseJSON is locked, but "untyped object" is translated.